egs/gale_arabic/s5c/local/chain/compare_wer.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
Is this file exactly the same as the one from s5b? @danpovey I don't know what's the convention here regarding different s5* recipes. Should we just link to the old scripts if files are identical?